dll/cpl/timedate/internettime.c

```diff
@@ -93,12 +93,40 @@ SetNTPServer(HWND hwnd)
     UINT uSel;
     WCHAR szSel[4];
     LONG lRet;
+    WCHAR buffer[256];
 
     hList = GetDlgItem(hwnd,
                        IDC_SERVERLIST);
 
     uSel = (UINT)SendMessageW(hList, CB_GETCURSEL, 0, 0);
 
+    SendDlgItemMessage (hwnd, IDC_SERVERLIST, WM_GETTEXT, 256 * sizeof(WCHAR), (LPARAM) buffer);
```

Comment:
Buffer overflow. (Dont multiply with sizeof WCHAR)

Reply:
_countof should do the trick.

Reply:
I will fix this. Thanks.